modules/GTFE_Goods.js

```diff
@@ -45,7 +45,8 @@ class GTFE_Goods {
     let classPrefix = undefined;
     let goods = undefined;
     let tgtNodeId = undefined;
-    if (cardType['type'] == 'planets') {
+    const CARD_PLANETS = 8;
```

Comment:
Again... defining this one here and CARD_METEORIC_SWARM in GTFE_Card is confusing / inconsistent